### PR TITLE
fix(remix): Deprecate `loadUser`, `loadSession` and `loadOrganization` on rootAuthLoader

### PR DESCRIPTION
**Pull request by anagstef**
Currently, `rootAuthLoader` types suggest the usage of `loadUser`, `loadSession` and `loadOrganization` boolean options. Using any of these will result to a call to Clerk Backend API on each root loader execution. This results in rate limiting issues for apps with many users and affects scalability.

This PR deprecates these 3 options so that they're not misused.

Relevant issue: #1043 

## Checklist

- [x] `npm test` runs as expected.
- [x] `npm run build` runs as expected.
- [ ] (If applicable) [JSDoc comments](https://jsdoc.app/about-getting-started.html) have been added or updated for any package exports
- [ ] (If applicable) [Documentation](https://github.com/clerk/clerk-docs) has been updated

## Type of change

- [x] 🐛 Bug fix
- [ ] 🌟 New feature
- [ ] 🔨 Breaking change
- [ ] 📖 Refactoring / dependency upgrade / documentation
- [ ] other:

### CODE REVIEW

**Review by octoper — APPROVED** [review]
🔥

**Comment by panteliselef** [comment]
@anagstef @octoper I feel like these should have been removed with the Core 2 launch, was this an oversight on our part ?

**Comment by anagstef** [comment]
@panteliselef Yes, we should have removed them.

File: .changeset/hip-pets-run.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/remix': patch
+---
+
+Deprecate loadUser, loadSession and loadOrganization on rootAuthLoader options
```
Comment (desiprisg) [inline .changeset/hip-pets-run.md:5]:
```suggestion
Deprecate `loadUser`, `loadSession` and `loadOrganization` on `rootAuthLoader` options
```

**Review by desiprisg — APPROVED** [review]

**Comment by dimkl** [comment]
I think we should also use the `import { deprecated } from "@clerk/shared"` to add a deprecation warning in the server logs.
cc: @anagstef

**Pull request merged**